.github/workflows/semantic-release.yml

```diff
@@ -0,0 +1,65 @@
+name: Release
+
+on:
+  push:
+    branches:
+    - master
+    paths-ignore:
+    - "nuxt/**"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    outputs:
+      new_release_version: ${{ steps.semantic.outputs.new_release_version }}
+      new_release_published: ${{ steps.semantic.outputs.new_release_published }}
+  
+    steps:
+      - uses: actions/checkout@v3
+      - run: yarn
+      - uses: cycjimmy/semantic-release-action@e1fe1fc00a3729593e87efb2f88475de76d64a24
+        id: semantic
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+  publish:
+    needs: [release]
+    if: needs.release.outputs.new_release_published == 'true'
```

Comment:
Then, if a new release was created (based on commit messages) it will build the multi-platform artifacts and will add them to the Release.